.github/workflows/test.yml

```diff
@@ -11,12 +11,13 @@ jobs:
       run:
         shell: bash
     steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
     - name: Install Go
       uses: actions/setup-go@v5
       with:
         go-version: ${{ matrix.go-version }}
-    - name: Checkout code
-      uses: actions/checkout@v4
+        cache: false
```

Comment:
Do you think a comment would make sense here, to briefly explain why we're disabling cache?

Not a blocker for sure; just thinking out loud.

Reply:
I usually rely on git blame and commit messages, but a short note wouldn't hurt.

Reply:
added